dogu.json

```diff
@@ -0,0 +1,81 @@
+{
+  "Name": "k8s/nginx-static",
+  "Version": "1.23.1-0",
+  "DisplayName": "Nginx Static",
+  "Description": "Nginx Static is a webserver responsible to provide static content in the Cloudogu EcoSystem.",
```

Comment:
You could mention that this is a webserver for multinode.

Reply:
Done